editor.planx.uk/src/@planx/components/Confirmation/Public.tsx

```diff
@@ -40,6 +68,23 @@ export default function ConfirmationComponent(props: Props) {
     }
   });
 
+  return (
+    <Presentational
+      {...props}
+      applicableDetails={applicableDetails}
+      data={data}
+      sessionId={sessionId}
+    />
+  );
+}
+
+interface PresentationalProps extends Props {
+  sessionId: string;
+  applicableDetails: Record<string, string>;
+  data: QuestionAndResponses[];
+}
+
+export function Presentational(props: PresentationalProps) {
```

Comment:
The addition of a `Presentational` component here is purely a convenience so we can continue mocking `applicableDetails` for Storybook (eg similar to Review page which is another component that relies heavily on store data). Could just as easily return this via `Confirmation` directly, but then Storybook would only ever display a single row `"Application reference"` for details section as that's gauranteed to always be in state.